recipes/atari-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "atari-py" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6249ad5079b0489e87eb44e65485bb1b07cc1b5af729f1ee52ece749503ceb1d
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - pip
+    - python
+    - zlib
+  run:
+    - python
+    - numpy
+    - six
+    - zlib
+
+test:
+  imports:
+    - atari_py
+  requires:
+    - nose2
+
+about:
+  home: https://github.com/openai/atari-py
+  license: GPL-2
+  license_family: GPL
+  license_file: License.txt
```

Comment:
Since you've manually packaged the `license_file`, please submit an issue or a PR upstream to add the license file to the source distribution (probably through the addition of a `MANIFEST.in` file and appropriate entry. Then put the link to the issue/PR as a comment here as a reminder to check to see if the license is included in future releases so the local copy can be removed.

Reply:
@synapticarbors the license is packaged in the GitHub repo, just not the PyPi source distribution. I need to make an issue on the repo for the PyPi distribution and link that in my recipe?